yarn-project/end-to-end/src/fixtures/snapshot_manager.ts

```diff
@@ -414,6 +416,19 @@ async function setupFromFresh(
     writeFileSync(`${statePath}/aztec_node_config.json`, JSON.stringify(aztecNodeConfig));
   }
 
+  // If initial validators are provided, we need to remove them from the node.
```

Comment:
Should it state that you will only remove the first, e.g., just that if multiple initial is set you don't remove them all

Reply:
ah good spot this is legacy, since in the e2e test fix i added a little pr to remove the initial validator only if you want to, will remove this